mongodbatlas/resource_mongodbatlas_advanced_cluster_migration_test.go

```diff
@@ -1,132 +0,0 @@
-package mongodbatlas
-
-import (
-	"fmt"
-	"testing"
-
-	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
-)
-
-func TestAccClusterRSAdvancedClusterMigrateState_empty_advancedConfig(t *testing.T) {
```

Comment:
this file was not deleted but renamed to resource_mongodbatlas_advanced_cluster_test.go since it has unit tests and not acceptance tests